dubbo-filter/dubbo-filter-cache/src/main/java/com/alibaba/dubbo/cache/support/AbstractCacheFactory.java

```diff
@@ -30,7 +32,8 @@
 
     private final ConcurrentMap<String, Cache> caches = new ConcurrentHashMap<String, Cache>();
 
-    public Cache getCache(URL url) {
+    public Cache getCache(URL url, Invocation invocation) {
+        url = url.addParameter(Constants.METHOD_KEY, invocation.getMethodName());

```

Comment:
can you attach a testcase for these modified codes, thx.

Reply:
ok, i need some help because i don't know how to test.
l debug in idea run ok.

Reply:
you can mock an invocation and cacheFactory,
then assert the result hitting your cache.

Reply:
how to confirm use annother cache not lru

Reply:
The change looks good, except for this:

```java
url = url.addParameter(Constants.METHOD_KEY, invocation.getMethodName());
```
I don't think we need addParameter for method name in getCache() method